environment.yml

```diff
@@ -12,6 +12,10 @@ dependencies:
   - cpuonly
   - pytorch
   - torchvision
+  - opencv
```

Comment:
Can we remove `opencv` from the conda dependencies and put `opencv-python-headless` under the pip part? The normal one won't run on the EC2 instances because it includes a bunch of GUI related stuff that the headless version doesn't.